Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -455,9 +455,6 @@ void CVolumeDirectory::WriteDirectory(const File::FSTEntry& parent_entry, u32* f
   // Sort for determinism
   std::sort(sorted_entries.begin(), sorted_entries.end(), [](const File::FSTEntry& one,
                                                              const File::FSTEntry& two) {
-    // For some reason, sorting by lowest ASCII value first prevents many games from
-    // fully booting. We make the comparison case insensitive to solve the problem.
-    // (Highest ASCII value first seems to work regardless of case sensitivity.)
```

Comment:
Please remove this in the commit that fixes the off by one issue instead of this commit, so people who look at the commit history without looking at the PR will understand why the comment was removed.